css/properties.json

```diff
@@ -3374,6 +3374,21 @@
     ],
     "status": "standard"
   },
+  "color-adjust": {
+    "syntax": "economy | exact",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Color"
+    ],
+    "initial": "economy",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "perGrammar",
+    "status": "nonstandard"
```

Comment:
Should it be `standard` or `nonstandard` if its part of the CSS Color Module Level 4
Editor’s Draft, 20 March 2018?

Reply:
I'd say standard.

Reply:
Done!